CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 Changelog
 ===============================================================================
 
+## Version 4.0.0 - 2022-01-14
```

Comment:
Do you want to add changes from #595?

Should we add a note somewhere about the changes in morphometrics from #589?

Reply:
Yes for 589, no for 595 (just tests). On it.

Reply:
Done in [dea20b0](https://github.com/neuropoly/axondeepseg/pull/587/commits/dea20b01e494866e815295527385b4a028748390)